tests/integrate/Autotest.sh

```diff
@@ -11,7 +11,7 @@ stress_threshold=0.001
 # check accuracy
 ca=8
 # regex of case name
-case="^[^#].*_.*$"
+case='^[^#].*_.*(?<!GPU)$'
```

Comment:
@dzzz2001 After consideration, I think adding GPU test cases by default and excluding them by default regex may add confusion to users who does not requires GPU test cases. If developers want to run test by a specific case name pattern, they are required to exclude GPU test cases manually.
In my humble opinion, keeping the existed test cases unchanged and adding GPU test cases into another CASE file would be better. What do you think?

Reply:
@caic99 Thanks for your well-considered suggestion, and I fully agree with it. I plan to put the GPU test filenames in a new file named CASES_GPU, and add a new parameter -f to the Autotest.sh script to specify the file containing the test filenames. By doing so, we can differentiate between GPU and CPU test cases using filenames rather than regex. What do you think about doing it this way?

Reply:
LGTM, and how about adding .txt suffix to that files?